app/uk/gov/hmrc/helptosavetestadminfrontend/connectors/AuthConnector.scala

```diff
@@ -34,11 +34,10 @@ import scala.annotation.tailrec
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Random
 
-class AuthConnector @Inject() (http: HttpClient, appConfig: AppConfig) extends Logging {
-
+class AuthConnector @Inject() (http: HttpClient, appConfig: AppConfig)(implicit ec: ExecutionContext) extends Logging {
```

Comment:
just out of curiosity, why?

Reply:
it's only needed at class level, it is not request specific, it is cleaner